src/components/Header/Header.tsx

```diff
@@ -34,97 +30,120 @@ export interface HeaderProps {
   isProduction?: boolean;
 }
 
+/**
+ * The useMediaQuery hook returns an array of booleans,
+ * indicating whether the given query matches or queries match.
+ * */
+function useMediaQuery(queryList) {
```

Comment:
Pull this out into `headerUtils`? It is not used anywhere else

Reply:
Yes, please. I missed this in the v3 release so it should be exported by the DS so you and other teams don't have to manually copy the function into a utils file.